web/src/pages/ProfileEditPage.vue

```diff
@@ -24,7 +24,15 @@
       </div>
     </h2>
 
+    <AdminUserEditForm
+      v-if="isSystemAdmin"
+      class="mt-10"
+      :user-id="currentUser.id"
+      :cancel-button-options="{ to: { name: 'ProfilePage' } }"
+      @saved="refresh"
+    />
```

Comment:
I think I mentioned this before, but maybe didn't explain it properly. I don't think the profile is the correct place for an Admin edit interface.

I would expect all users to have the same field editability for their profile, and if the admin wants to edit a user's more restricted or complex details, they would go to the user admin interface; even if that user is them.